apps/src/templates/codeDocs/ProgrammingClassOverview.jsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import EnhancedSafeMarkdown from '@cdo/apps/templates/EnhancedSafeMarkdown';
+import Example from './Example';
+import FieldsTable from './FieldsTable';
+import i18n from '@cdo/locale';
+
+export default function ProgrammingClassOverview({programmingClass}) {
+  return (
+    <div style={{width: '100%'}}>
+      <h1>{programmingClass.name}</h1>
+      {programmingClass.category && (
+        <div>
+          <strong>{`${i18n.category()}:`}</strong>
+          <span
+            style={{
+              backgroundColor: programmingClass.color,
+              marginLeft: 10,
+              padding: '5px 10px'
+            }}
+          >
+            {programmingClass.category}
+          </span>
+        </div>
+      )}
+      {!!programmingClass.content && (
+        <div style={{paddingTop: 20}}>
+          <EnhancedSafeMarkdown
+            markdown={programmingClass.content.trim()}
+            expandableImages
+          />
+        </div>
+      )}
+      {programmingClass.examples?.length > 0 && (
+        <div>
+          <h2>Examples</h2>
```

Comment:
should this be translatable? Same for Fields below

Reply:
Oh good catch! I'll fix here and try to remember to do a pass of other spots I might've forgotten to do this...